README.md

```diff
@@ -37,6 +37,7 @@ Fingerprint Pro React SDK is an easy way to integrate **[Fingerprint Pro](https:
 ## Table of contents
 
 - [Installation](#installation)
+- [Requirements](#requirements)
```

Comment:
Please align the intro with our template: https://www.notion.so/fingerprintjs/Writing-DX-GitHub-Readmes-WIP-9556ed587c0749b29f32a18017413cbc?pvs=4#ee9bf8b126bd4ccb8e91ff20b1fc70aa

Reply:
Done!